### PR TITLE
843 arv medication section

### PR DESCRIPTION
**Pull request by clemens-msupply**
Implements some of the low hanging fruits from #843 

- Also did some style and alignment updates for various select and autocomplete boxes... (opinion welcome)
- Also pass on enable flags to various components (needed for JSONForms rules)

### CODE REVIEW

File: server/service/src/sync/program_schemas/hiv_care_encounter_ui_schema.json [hunk 1]
```diff
@@ -141,76 +128,68 @@
...
-              ["FIRST_LINE_ADULT", "First Line - Adult"],
-              ["FIRST_LINE_PAEDS_20_30_KG", "First Line - Paeds 20-30kg"],
-              ["FIRST_LINE_PAEDS_0_20_KG", "First Line - Paeds <20kg"],
-              ["SECOND_LINE_ADULT", "Second Line - Adult"],
-              ["SECOND_LINE_PAEDS_20_30_KG", "Second Line - Paeds 20-30kg"],
-              ["SECOND_LINE_PAEDS_0_20_KG", "Second Line - Paeds <20kg"]
+              ["NO", "No ARV"],
+              ["START", "Start ARV"],
+              ["CONTINUE", "Continue ARV"],
+              ["CHANGE", "Changes to ART Regimen"],
```
Comment (roxy-dao) [inline server/service/src/sync/program_schemas/hiv_care_encounter_ui_schema.json:156]: ARV..?

**Review by roxy-dao — APPROVED** [review]
Looks good~ 

Just a side note while going through this... the NumericTextInput should probably be NonNegativeNumberInput 🤔 since a bit weird to have negative pill count or prescriptions

File: server/service/src/sync/program_schemas/hiv_care_encounter_ui_schema.json (hunk 1 above)
Comment (clemens-msupply) [reply server/service/src/sync/program_schemas/hiv_care_encounter_ui_schema.json:156]: thanks fixed it

**Comment by clemens-msupply** [comment]
I added some more change to use a PositiveNumberInput if the minimum is >= 0. Think this catches the most common case to have positive inputs...

**Comment by clemens-msupply** [comment]
sorry just noticed I did the changes in the follow up branch 598-multi-column-select-option...

**Pull request merged**